### ISSUE TITLE
Contract's Storage is empty

### ISSUE THREAD

**Issue by yhguan123** [trufflesuite/ganache#1200]
There are many properties in Contract's Storage is empty. what's the problem? Ganache v2.5.4

**Comment by milkiweed** [comment]
same issue. In my case ganache can only display uint variables, mapping always remains empty

OS Version: macOS version 12.0 Beta
Genache: Version 2.5.4 (2.5.4.1367)

**Comment by yhguan123** [comment]
> same issue. In my case ganache can only display uint variables, mapping always remains empty
> 
> OS Version: macOS version 12.0 Beta Genache: Version 2.5.4 (2.5.4.1367)

yes,maybe the truffle suite is not in maintaining from now on. we should change the framework to hardhat.
